code/mob/living/carbon/human.dm

```diff
@@ -1375,6 +1375,44 @@
 		return r_store
 	return null
 
+/mob/living/carbon/human/get_slot_from_item(var/thing)
```

Comment:
thing can be typed to be an obj/item like you do in the original definition

Reply:
will do, for some reason I had a change of heart about "what if the equipped thing is an object instead of an item" halfway through